app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,107 @@
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:inputType="number"
+            android:hint="@string/profile_height_prompt"/>
+        <EditText
+            android:id="@+id/height_inches_field"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:inputType="number"
+            android:hint="@string/profile_height_inches_prompt"/>
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:paddingStart="@dimen/profile_padding_horizontal_labels"
+        android:paddingEnd="@dimen/profile_padding_horizontal_labels" >
+        <TextView
+            android:textSize="@dimen/profile_label_text_size"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:text="@string/profile_height_feet_units"
+            android:labelFor="@+id/height_feet_field" />
+        <TextView
+            android:textSize="@dimen/profile_label_text_size"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:paddingStart="@dimen/profile_padding_horizontal_labels"
+            android:singleLine="true"
+            android:text="@string/profile_height_inches_units"
+            android:labelFor="@+id/height_inches_field" />
+    </LinearLayout>
+
+    <EditText
+        android:id="@+id/weight_field"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="number"
+        android:hint="@string/profile_weight_prompt"/>
+    <TextView
+        android:textSize="@dimen/profile_label_text_size"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="@dimen/profile_padding_horizontal_labels"
+        android:text="@string/profile_weight_lbs_units"
+        android:labelFor="@+id/weight_field" />
+
+    <!--<TextView-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:text="@string/profile_dietary_restrictions_prompt" />-->
+    <!--<android.support.v7.widget.RecyclerView-->
+        <!--android:id="@+id/profile_recycler_view"-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"> </android.support.v7.widget.RecyclerView>-->
+    <!--<Button-->
+        <!--android:id="@+id/diet_button"-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:text="@string/profile_dietary_restrictions_button_text"/>-->
```

Comment:
Please kill this with fire

Reply:
(but write it down somewhere so you can restore it)

Reply:
:fire: